T/TempestModel/build_tarballs.jl

```diff
@@ -0,0 +1,111 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TempestModel"
+version = v"0.1"
+sources = [
+    GitSource("https://github.com/paullric/tempestmodel",
+	      "4ecf5146f3100fb24d36156a2f99433f049f0f66"),
+    DirectorySource("./bundled")
+]
+
+script = raw"""
+mkdir -vp $bindir
+mkdir -vp $libdir
+
+cd ${WORKSPACE}/srcdir/tempestmodel*
+
+atomic_patch -p1 ../patches/mk_defs.patch  
+atomic_patch -p1 ../patches/mk_system_macosx.patch
+# linux system fallback
+atomic_patch -p1 ../patches/mk_system_agri.patch
+atomic_patch -p1 ../patches/netcdf_cpp.patch
+
+export CXXFLAGS="-I${includedir}"
+export LDFLAGS="-L${libdir}"
+
+make NETCDF_ROOT=$prefix -j${nproc} all
+
+cp -v test/nonhydro_sphere/ScharMountainSphereTest ${bindir}
+cp -v test/nonhydro_sphere/BaroclinicWaveJWTest ${bindir}
+cp -v test/nonhydro_sphere/StationaryMountainFlowTest ${bindir}
+cp -v test/nonhydro_sphere/MountainWaveSphereTest ${bindir}
+cp -v test/nonhydro_sphere/HeldSuarezTest ${bindir}
+cp -v test/nonhydro_sphere/InertiaGravityWaveTest ${bindir}
+cp -v test/nonhydro_sphere/BaroclinicWaveUMJSTest ${bindir}
+cp -v test/nonhydro_sphere/MountainRossby3DTest ${bindir}
+cp -v test/nonhydro_sphere/BaldaufGravityWaveTest ${bindir}
+
+cp -v test/nonhydro_xz/RobertBubbleCartesianTest ${bindir}
+cp -v test/nonhydro_xz/ShearJetMtnWave2DCartesianTest ${bindir}
+cp -v test/nonhydro_xz/ThermalBubbleCartesian3DTest ${bindir}
+cp -v test/nonhydro_xz/Baroclinic3DCartesianTest ${bindir}
+cp -v test/nonhydro_xz/DensityCurrentCartesianTest ${bindir}
+cp -v test/nonhydro_xz/NonHydroMountainCartesianTest ${bindir}
+cp -v test/nonhydro_xz/HydrostaticMountainCartesianTest ${bindir}
+cp -v test/nonhydro_xz/ThermalBubbleCartesianTest ${bindir}
+cp -v test/nonhydro_xz/Baroclinic3DCartesianRidgeTest ${bindir}
+cp -v test/nonhydro_xz/ScharMountainCartesianTest ${bindir}
+cp -v test/nonhydro_xz/InertialGravityCartesianXZTest ${bindir}
+
+cp -v test/shallowwater_sphere/BarotropicInstabilityTest ${bindir}
+cp -v test/shallowwater_sphere/MountainRossbyTest ${bindir}
+cp -v test/shallowwater_sphere/RossbyHaurwitzWaveTest ${bindir}
```

Comment:
Fully automated installation, I see

Reply:
I did automate the string formation to paste in here with julia lol

Reply:
Yeah, sorry I was joking :slightly_smiling_face: